targets/arm32/cm3/atmel/sam3x/sam3x8e/flashscripts/arduino_due.cmake

```diff
@@ -9,17 +9,17 @@ set(DEVICE_LOCATION
 )
 
 if(bossac)
-    if(WIN32)
+    if(MSYS OR MINGW)
```

Comment:
Checking for win32 checks the build target and since it's compiling for arduino win32 will never be defined.